README.md

```diff
@@ -41,6 +41,68 @@ from memphis.types import Retention, Storage
 import asyncio
 ```
 
+### Quickstart - Producing and Consuming
+
+The most basic functionaly of memphis is the ability to produce messages to a station and to consume those messages. 
+
+> The Memphis.py SDK uses asyncio for many functions. Make sure to call the following code in an async function:
+
+```python 
+async def main():
+    ...
+
+if __name__ == '__main__':
+  asyncio.run(main()) 
+```
+
+First, a connection to Memphis must be made:
+
+```python
+from memphis import Memphis
+
+# Connecting to the broker
+memphis = Memphis()
+
+await memphis.connect(
+    host = "<memphis-host>",
+    username = "<memphis-username>",
+    password = "<memphis-password>",
+    account_id = <mmephis-account-id> # For cloud users, at the top of the overview page
```

Comment:
memphis

Reply:
@shay23b Same issue with the comment in the C# branch, where this fails linting when I don't comment out that. 

Is that okay? I don't know how to change the rules so it allows that.